code/__HELPERS/text.dm

```diff
@@ -449,7 +449,7 @@
 
 
 // Pencode
-/proc/pencode_to_html(text, mob/user, obj/item/pen/P = null, format = 1, sign = 1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT, no_font = FALSE)
+/proc/pencode_to_html(text, mob/user, obj/item/pen/P = null, format = 1, sign = 1, name=1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT, no_font = FALSE)
```

Comment:
1. Put new arguments at the end unless you want to change all the code that uses `pencode_to_html`
2. Make sure theres a space around the `=`
3. Use TRUE instead of 1, we have a lot of places in the codebase that uses "1" but we're trying to transition to TRUE
```suggestion
/proc/pencode_to_html(text, mob/user, obj/item/pen/P = null, format = 1, sign = 1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT, no_font = FALSE, name = TRUE)
```

Reply:
Ah okay, I was just trying to follow the "Format, sign, fields" because I put my code kinda inbetween them so thought I should use what they used.